tools/jenkins/run_full_performance.sh

```diff
@@ -52,4 +52,12 @@ tools/run_tests/run_performance_tests.py \
     --remote_worker_host grpc-performance-server-32core grpc-performance-client-32core grpc-performance-client2-32core \
     || EXIT_CODE=1
 
+# selected scenarios on Windows
```

Comment:
Currently making it run as a part of a different job is problematic because when the data is stored to bigquery, the name of the jenkins job is stored as one of the fields.
Our dashboard then includes jobName as part of the queries that are used to render the charts.

Example:
```
SELECT
	...
FROM
	[grpc-testing:performance_test.performance_experiment]
WHERE
	metadata.created >= DATE_ADD(CURRENT_TIMESTAMP(), -1, "WEEK") AND
    metadata.jobName = "gRPC_performance_experiment" AND
   ...
```